Platform/Xiaomi/sdm660/whyred.dsc

```diff
@@ -0,0 +1,24 @@
+[Defines]
+  PLATFORM_NAME                  = whyred
+  PLATFORM_GUID                  = 28f1a3bf-193a-47e3-a7b9-5a435eaab2ee
+  PLATFORM_VERSION               = 0.1
+  DSC_SPECIFICATION              = 0x00010019
+  OUTPUT_DIRECTORY               = Build/$(PLATFORM_NAME)
+  SUPPORTED_ARCHITECTURES        = AARCH64
+  BUILD_TARGETS                  = DEBUG|RELEASE
+  SKUID_IDENTIFIER               = DEFAULT
+  FLASH_DEFINITION               = Platform/Qualcomm/sdm660/sdm660.fdf
+  DEVICE_DXE_FV_COMPONENTS       = Platform/Xiaomi/sdm660/whyred.fdf.inc
+
+!include Platform/Qualcomm/sdm660/sdm660.dsc
+
+[BuildOptions.common]
+  GCC:*_*_AARCH64_CC_FLAGS = -DENABLE_SIMPLE_INIT
+
+[PcdsFixedAtBuild.common]
+  gQcomTokenSpaceGuid.PcdMipiFrameBufferWidth|1080
+  gQcomTokenSpaceGuid.PcdMipiFrameBufferHeight|2160
+
+  gRenegadePkgTokenSpaceGuid.PcdDeviceVendor|"Redmi"
+  gRenegadePkgTokenSpaceGuid.PcdDeviceProduct|"5"
+  gRenegadePkgTokenSpaceGuid.PcdDeviceCodeName|"whyred"
```

Comment:
using simple-init but missing PcdGuiDefaultDPI?

Reply:
Oh, I think I ignored it. I will solve it immediately